sdk/servicebus/service-bus/src/core/batchingReceiver.ts

```diff
@@ -495,7 +487,7 @@ export class BatchingReceiverLite {
       };
 
       abortSignalCleanupFunction = checkAndRegisterWithAbortSignal((err) => {
-        cleanupBeforeResolveOrReject("removeDrainHandler");
+        cleanupBeforeResolveOrReject();
```

Comment:
Is this still a place where we may lose messages?

- abort signal is invoked
- we remove the abort listener
- we call the cleanup 
- cleanup remove the message listener first at the time where there may be active messages coming in?

Reply:
I'll create a follow up issue for that one.

Reply:
Issue: https://github.com/Azure/azure-sdk-for-js/issues/12922